engine/classes/Elgg/Application/CacheHandler.php

```diff
@@ -83,7 +83,7 @@ public function handleRequest($path) {
 			exit;
 		}
 
-		$filename = $config->getVolatile('dataroot') . 'views_simplecache/' . md5("$viewtype|$view");
```

Comment:
I don't follow, how is it that we can have long views in the views directory but then when we cache them they have to be shortened?